meta/main.yml

```diff
@@ -0,0 +1,72 @@
+# SPDX-License-Identifier: MIT
+---
+galaxy_info:
+  # Replace with role's author name:
+  author: John Doe <jdoe@corp.com>
```

Comment:
remove the comments and fill in the fields with the actual data

Reply:
https://github.com/linux-system-roles/certificate/pull/1/commits/b624c0071a6c1ff291c066cd99df9e4e7a4f1c75